### PR TITLE
Edit blur events

### PR DESCRIPTION
**Pull request by daintycode**

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -27,6 +27,30 @@ injectGlobal`
 		line-height: 1.5;
 		margin: 0;
 	}
+
```
Comment (k15a) [inline src/components/App/App.js]: You could move this to the `CodeEditor.js` file so it's collocated.

File: src/components/Playground/Editor/Panels/CodeEditor.js [hunk 2]
```diff
@@ -49,28 +50,40 @@ class Editor extends Component {
...
+					this.element.refs.editor.classList.remove('blurred');
+				}}
+				onBlur={() => {
+					this.setState({
+						isFocused: false,
+					});
+
+					this.element.refs.editor.classList.add('blurred');
+				}}
+				className="blurred"
```
Comment (k15a) [inline src/components/Playground/Editor/Panels/CodeEditor.js]: I think className={this.state.isFocues ? "blurred" : ""} would be better ;)

File: src/components/Playground/Editor/Panels/Output.Container.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { connect } from 'react-redux';
 import Output from './Output.js';
 
 const mapStateToProps = ({ markup, script, style }) => ({
-	markup: markup.code,
+	markup: markup.compiled,
 	script: script.compiled,
 	style: style.code,
```
Comment (k15a) [inline src/components/Playground/Editor/Panels/Output.Container.js]: Should be `style.compiled` :p

File: src/components/Playground/Editor/Panels/CodeEditor.js (hunk 2 above)
Comment (daintycode) [inline src/components/Playground/Editor/Panels/CodeEditor.js]: ace goes completely crazy when doing this, may need to stay like this.^^

**Pull request merged**